freezegun/api.py

```diff
@@ -727,6 +727,7 @@ def freeze_time(time_to_freeze=None, tz_offset=0, ignore=None, tick=False, as_ar
     ignore.append('selenium')
     ignore.append('_pytest.terminal.')
     ignore.append('_pytest.runner.')
+    ignore.append('_pytest.runner')
```

Comment:
`test_should_use_real_time` fails. But if I comment out the lines that change `call_stack_inspection_limit`, they pass.

As a sanity check, I undid my change, left the `call_stack_inspection_limit` lines commented out, and the tests still passed. So I'm not sure what this is doing, other than the comment about increasing coverage.

Reply:
And it looks like the other tests fail because this value never gets reset.